easybuild/easyblocks/generic/pythonbundle.py

```diff
@@ -91,6 +91,10 @@ def prepare_step(self, *args, **kwargs):
         # when system Python is used, the first 'python' command in $PATH will not be $EBROOTPYTHON/bin/python,
         # since $EBROOTPYTHON is set to just 'Python' in that case
         # (see handling of allow_system_deps in EasyBlock.prepare_step)
+        req_py_majver = None
+        req_py_minver = None
+        max_py_majver = None
+        max_py_minver = None
```

Comment:
I don't think it's necessary (anymore) to initialize these variables? They're only being used in the else branch of the if statement, and all of them are initialized using self.cfg in there.
```suggestion
```

Reply:
@casparvl confirmed on Slack that this can be removed, so I'm committing the suggested change.